modules/visual_script/visual_script.h

```diff
@@ -60,7 +61,7 @@ class VisualScriptNode : public Resource {
 	static void _bind_methods();
 
 public:
-	Ref<VisualScript> get_visual_script() const;
+	Ref<Resource> get_container() const;
```

Comment:
Is Resource the correct type?

Reply:
As I was discussing last time, we could make a base type that allows us to do it, I could even use VisualScript itself as the base type for submodules but at the moment I am not sure because I am not sure all the features that we should add to submodules from visualscript.
That's something that I will do once we get to finalizing.